README.md

```diff
@@ -317,13 +317,16 @@ When no time period is specified, the query defaults to the last 2 days:
 
 You can view requests and responses statistics
  - for the `/analyze` endpoint at http://localhost:8080/metrics/analyze
- - for the `/analyze/staged` endpoint at http://localhost:8080/metrics/analyze/staged.
+ - for the `/analyze-staged` endpoint at http://localhost:8080/metrics/analyze-staged.
+ - for the requests coming from gitlab: http://localhost:8080/metrics/analyze-gitlab.
 
 You can retrieve single svg images at the following endpoints:
- - `/metrics/analyze/requests`
- - `/metrics/analyze/responses`
- - `/metrics/analyze/staged/requests`
- - `/metrics/analyze/stages/responses`
+ - http://localhost:8080//metrics/analyze/requests
+ - http://localhost:8080//metrics/analyze/responses
+ - http://localhost:8080//metrics/analyze-staged/requests
+ - http://localhost:8080//metrics/analyze-staged/responses
+ - http://localhost:8080//metrics/analyze-gitlab/requests
+ - http://localhost:8080//metrics/analyze-gitlab/responses
```

Comment:
```suggestion
 - http://localhost:8080/metrics/analyze/requests
 - http://localhost:8080/metrics/analyze/responses
 - http://localhost:8080/metrics/analyze-staged/requests
 - http://localhost:8080/metrics/analyze-staged/responses
 - http://localhost:8080/metrics/analyze-gitlab/requests
 - http://localhost:8080/metrics/analyze-gitlab/responses
```

I just realized this one extra leading `/` causes 404 :/ can you please correct these links?